admin/views/test-developer/view-check-form.ejs

```diff
@@ -18,51 +7,61 @@ let num = 1;
-<br />
+        <div class="panel panel-border-wide">
+            <% if (form.checkWindowsName.length > 0) { %>
+            This form is assigned to: <%= form.checkWindowsName %>
+            <% } else { %>
+            This form is unassigned
+            <% } %>
+        </div>
 
-<div class="grid-row">
-    <div class="column-half">
-        <dl class="inline-definition-list" tabindex="0" role="list">
-            <% col1.forEach((q) => { %>
-            <dt class="inline-definition-list" tabindex="0" role="listitem">Q<%= num++ %></dt>
-            <dd class="inline-definition-list">
-                <span class="fixed-factors" tabindex="0" role="listitem"><%= q.f1 %> &times; <%= q.f2 %></span>
-                <span class="equals-sign" tabindex="0" role="listitem">=</span>
-                <span class="fixed-result" tabindex="0" role="listitem"><%= (q.f1 * q.f2) %></span>
-            </dd>
-            <% }); %>
-        </dl>
+        <div class="padding-top">
+            <dl class="inline-definition-list" role="list">
+                <% form.questions.forEach((q) => { %>
+                <dt class="inline-definition-list" role="listitem">Q<%= num++ %></dt>
+                <dd class="inline-definition-list">
+                    <span class="first-factor" role="listitem"><%= q.f1 %></span>
+                    <span class="times-sign" role="listitem">&times;</span>
+                    <span class="second-factor" role="listitem"><%= q.f2 %></span>
+                    <span class="equals-sign" role="listitem">=</span>
+                    <span class="fixed-result" role="listitem"><%= (q.f1 * q.f2) %></span>
+                </dd>
+                <% }); %>
+            </dl>
+            <% if (form.canDelete === true) { %>
+            <br />
+            <a href="/test-developer/delete-form/<%= form._id %>" class="modal-link" id="js-modal-link">Remove form</a>
+            <% } %>
+        </div>
+        <div class="back-to-top-container">
+            <a href="#top">Back to top</a>
+        </div>
     </div>
-    <div class="column-half">
-        <dl class="inline-definition-list">
-            <% col2.forEach((q) => { %>
-            <dt class="inline-definition-list">Q<%= num++ %></dt>
-            <dd class="inline-definition-list">
-                <span class="fixed-factors"><%= q.f1 %> &times; <%= q.f2 %></span>
-                <span class="equals-sign">=</span>
-                <span class="fixed-result"><%= (q.f1 * q.f2) %></span>
-            </dd>
-            <% }); %>
-        </dl>
+    <div class="column-one-third add-title-margin">
+        <aside role="complementary" class="support-column">
+            <h3 class="heading-medium">Related</h3>
+            <nav role="navigation">
+                <ul class="list">
+                    <li>
+                        <a href="/PDFs/MTC_administration_guidance_June-2017-trial.pdf" target="_blank">Guidance</a>
```

Comment:
Add a rel="noopener noreferrer"  attribute to any links that use  target="_blank" . This protects users against security vulnerabilities associated with the target="_blank"  attribute.

https://paper.dropbox.com/doc/Links-xiWNcJk5Wpb8Sj6G4Prtl

Reply:
:thumbsup: